src/LedgerArk.js

```diff
@@ -10,7 +10,7 @@ var LedgerArk = function(comm) {
 
 LedgerArk.prototype.getAddress_async = function(path) {
 	var splitPath = utils.splitPath(path);
-	var buffer = new Buffer(5 + 1 + splitPath.length * 4);
+	var buffer = Buffer.from(5 + 1 + splitPath.length * 4);
```

Comment:
If you wish to create a new Buffer of a specific length you need to use Buffer.alloc